doc/install/index.rst

```diff
@@ -1,62 +1,13 @@
-.. include:: ../links.inc
+.. _installers:
 
-.. _quick-start:
-
-Quick start
-===========
```

Comment:
I am tempted to keep this here and start with something like this:

```
If you’ve never worked with Python before, the simplest way to install MNE-Python is with the `installers`_. If you prefer to install manually jump our quick start instructions using the command line.
```

Then you insert here the RST for the installers which remains short with simple buttons. Below I would then have what you just moved below.

Reply:
I think it's ok to click on "Quick Start" and get installers. The page does not need to be called "Installers".

Reply:
Agreed. It would be good to keep the `quick-start` link, and have it point here